### PR TITLE
Remove Resolver Actor

### LINKED ISSUE #451: Remove resolver actor from actix
## Expected Behavior
actix as an actor framework should not be bind to a dns dependency.
I believe majority people don't use that feature or they would be fine to move that feature to user code base.

## Current Behavior
actix was blocked from update to tokio 1.0 because the said dep is still not ready.
At this point actix is unblocked but the point still stands.

## Possible Solution
Remove the feature.

### PR DESCRIPTION
**Pull request by Chaostheorie**
I'm sorry for the delay. I got sidetracked and forgot to commit and push my code.

## PR Type
Refactor / Removal of deprecated feature

## PR Checklist

- [x] Tests for the changes have been added / updated.
- [x] Documentation comments have been added / updated. (removed related docs)
- [x] A changelog entry has been made for the appropriate packages.
- [x] Format code with the latest stable rustfmt

## Overview
Removed actor and related docs.

Closes #451

### CODE REVIEW

File: actix/CHANGES.md [hunk 1]
```diff
@@ -20,6 +20,9 @@
 [#499]: https://github.com/actix/actix/pull/499
 [#501]: https://github.com/actix/actix/pull/501
 
+### Removed
+- Removed `Resolver` actor [#451]
+
```
Comment (robjtede) [inline actix/CHANGES.md:25]: add this to "unreleased"

**Review by robjtede — CHANGES_REQUESTED** [review]
would also need to remove the resolver feature in cargo toml and the trust-dns deps

File: actix/CHANGES.md (hunk 1 above)
Comment (Chaostheorie) [reply actix/CHANGES.md]: Changes were made in new commit. Thank you for the comment

**Comment by robjtede** [comment]
nice work, thanks 👍🏻

**Pull request merged**